plugin/references.py

```diff
@@ -191,3 +191,19 @@ def _group_references_by_file(self, references: 'List[ReferenceDict]'
         linecache.clearcache()
```

Comment:
Is it ok to leave the `linecache.clearcache()` function call here? :)

Reply:
I don't think it will cause any problems